graphql_compiler/compiler/expressions.py

```diff
@@ -928,11 +937,22 @@ def to_sql(self, dialect: Any, aliases: AliasesDictType, current_alias: AliasTyp
             # PostgreSQL
             # coalesce to an empty array of the corresponding type
             inner_type = strip_non_null_from_type(self.field_type.of_type)
-            if is_same_type(GraphQLInt, inner_type):
-                sql_array_type = "INT"
-            elif is_same_type(GraphQLString, inner_type):
-                sql_array_type = "VARCHAR"
-            else:
+            graphql_type_to_sql_array_type_dict = {
+                GraphQLInt: "INT",
+                GraphQLString: "VARCHAR",
+                GraphQLID: "UUID",
```

Comment:
This one isn't valid, unfortunately. The GraphQL `ID` type can map to SQL's various int and string types, as well as `UUID`. Probably safest to coerce to string.